lib/gitx/cli/nuke_command.rb

```diff
@@ -6,7 +6,7 @@ module Gitx
   module Cli
     class NukeCommand < BaseCommand
       desc 'nuke', 'nuke the specified aggregate branch and reset it to a known good state'
-      method_option :destination, :type => :string, :aliases => '-d', :desc => 'destination branch to reset to'
+      method_option :destination, type: :string, aliases: '-d', desc: 'destination branch to reset to'
```

Comment:
Line is too long. [102/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.